features/edit_menu.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Restaurant owner can update his menu
+  As a restaurant owner,
+  In order to keep my customers up to date regarding my products
+  I would like to be able to make updates to my menu
+  
+  Background: 
+    Given the following categories exists
+      | title    |
+      | Starters |
+    And the following products exists
+      | name          | description           | price | category |
+      | garlic bread  | garlic, bread, butter | 30    | Starters |
+    And the following user exists
+      | name | email        | password | admin |
+      | John | john@doe.com | password | true  |
+    And I visit the landing page
+    And I click "Login"
```

Comment:
can you use the warden helper to log you in instead of going thru all of these steps